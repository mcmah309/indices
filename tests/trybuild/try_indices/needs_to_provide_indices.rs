use indices::try_indices;

fn main() {
    let mut data: [i32; 0] = [];
    let slice = data.as_mut_slice();
    let result = try_indices!(slice);
}