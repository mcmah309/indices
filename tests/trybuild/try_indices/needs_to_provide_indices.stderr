error: unexpected end of macro invocation
 --> tests/trybuild/try_indices/needs_to_provide_indices.rs:6:36
  |
6 |     let result = try_indices!(slice);
  |                                    ^ missing tokens in macro arguments
  |
note: while trying to match `,`
 --> src/lib.rs
  |
  |     ($slice:expr, $( $index:expr ),+) => {{
  |                 ^
